src/python/foglamp/translators/omf_translator.py

```diff
@@ -178,7 +182,7 @@ def initialize_plugin():
 
     try:
         # URL
-        _server_name = "WIN-4M7ODKB0RH2"
+        _server_name = "192.168.0.221"
```

Comment:
Any reason to change that? its just one hard-coded value that works for someone and not for other. Ultimately all these values will be read from a foglamp system config file or configuration manager